packages/db-mongodb/src/create.ts

```diff
@@ -17,10 +17,16 @@ export const create: Create = async function create(
 
   const options: CreateOptions = {
     session: await getSession(this, req),
+    // Timestamps are manually added by the write transform
```

Comment:
I don't think this comment is adding much. I would remove it from all these operations.